presto-docs/src/main/sphinx/release/release-0.158.rst

```diff
@@ -7,3 +7,17 @@ General Changes
 
 * Fix regression that could cause high CPU and heap usage on coordinator,
   when processing certain types of long running queries.
+* Fix a bug in``EXPLAIN ANALYZE`` where output columns were incorrectly pruned.
```

Comment:
"Fix a bug" is bad phrasing as it's redundant (basically every fix is a bug fix). How about
```
* Fix incorrect pruning of output columns in ``EXPLAIN ANALYZE``.